src/components/PostList/PostList.jsx

```diff
@@ -2,0 +1,10 @@
+import { PostInfo } from '../PostInfo';
+import './PostList.scss';
+
+export const PostList = ({ posts }) => (
+  <div className="PostList">
+    {posts.map(post => (
+      <PostInfo post={post} key={post.id} />
```

Comment:
The `key` prop is correctly used here with `post.id`. Ensure that each post object has a unique `id` field to prevent rendering issues.